sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java

```diff
@@ -10483,21 +10478,17 @@ public int bm_revive_pet_1(obj_id self, obj_id target, String params, float defa
             setCommandTimerValue(self, TIMER_COOLDOWN, 0.0f);
             return SCRIPT_OVERRIDE;
         }
-        if (getDistance(getLocation(self), getLocation(beast)) > 7.0f)
-        {
-            sendSystemMessage(self, new string_id("spam", "stand_next_to_corpse"));
-            setCommandTimerValue(self, TIMER_COOLDOWN, 0.0f);
-            return SCRIPT_OVERRIDE;
-        }
         if (!combatStandardAction("bm_revive_pet_1", self, target, params, "", ""))
         {
             setCommandTimerValue(self, TIMER_COOLDOWN, 0.0f);
             return SCRIPT_OVERRIDE;
         }
+        float playerToPetRange = getDistance(getLocation(self), getLocation(beast));
+        float percentRangeMod = (playerToPetRange / 84f) * 100f;

```

Comment:
These two lines can be simplified to one: `float percentRangeMod = (getDistance(self, beast) / 84f) * 100f;` but it may be better to get the distance and store it as a variable at the beginning of this command so it can be referenced in the above if block for distance, as well as here and below.

Reply:
In this case, I would leave it as is for readability reasons.